### PR TITLE
Facets: Exclude facets on children of Tools.

### LINKED ISSUE #451: Hide Device facets on Tools ProductLists
It looks like https://github.com/iFixit/react-commerce/issues/375 was not implemented fully. Most facets were hidden on `/Tools` only but not in any of the Child ProuductLists. 

image

image

The different I see between the ProductLists is that [`/Tools` is `type: all-tools`](https://strapi.ifixit.com/admin/content-manager/collectionType/api::product-list.product-list/2226?plugins[i18n][locale]=en) and all of the ProductLists I've created and have set as [Children of `/Tools` are `type: tools`](https://strapi.ifixit.com/admin/content-manager/collectionType/api::product-list.product-list/2283?plugins[i18n][locale]=en). 

So, we need to hide the following facets on `type: tools`

`Device`
`Device Brand`
`Device Category`
`Device Type`
`Item Type`
`OS`
`Part or Kit`

And we should have included `Part or Kit` in the initial list of facets to hide. Please hide `Part or Kit` on `type: all-tools`.

CC @sterlinghirsh

### PR DESCRIPTION
**Pull request by sumukhipandey**
Previously, the children of Tools still displayed all the facets that were hidden on `/Tools`. This PR fixes that by keeping the excluded facets consistent across `/Tools` and its children.
`Part or Kit` is also added to the exclude list.

## QA
 Make sure that [tools children](https://github.com/iFixit/react-commerce/pull/510#issuecomment-1197235354) have the same facets as `/Tools`. [[https://tools--hide-facets-on-tools-children.govinor.com/admin](https://tools--hide-facets-on-tools-children.govinor.com/admin)]
   - I [added ](https://tools--hide-facets-on-tools-children.govinor.com/admin/content-manager/collectionType/api::product-list.product-list/1?plugins[i18n][locale]=en)`tools-child` to check the changes myself. If needed that can be viewed [here](https://react-commerce-git-tools-hide-facets-on-tools-children-ifixit.vercel.app/Tools/tools-child)

Closes #451 

CC: @erinemay @sterlinghirsh

### CODE REVIEW

File: frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx [hunk 1]
```diff
@@ -49,7 +49,13 @@ export function useFilteredFacets(productList: ProductList) {
       return usefulFacets;
    }, [facets, infoNames, sortBy]);
 
-   if (productList.type === ProductListType.AllTools) {
+   const isToolsPage =
+      productList.type === ProductListType.AllTools ||
+      productList.type === ProductListType.ToolsCategory
+         ? true
+         : false;
```
Comment (andyg0808) [inline frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx:56]: Pretty sure you should always get booleans from `===` anyhow:
```suggestion
      productList.type === ProductListType.AllTools ||
      productList.type === ProductListType.ToolsCategory;
```

**Review by andyg0808 — DISMISSED** [review]

**Comment by andyg0808** [comment]
CR :dog: through b3a342e
deploy_block :+1: on one suggestion.

File: frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx (hunk 1 above)
Comment (sumukhipandey) [reply frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx]: Resolved b33fc556fa304945d7ec0c7ea2594acc32e3ff6e

**Comment by sumukhipandey** [comment]
un_deploy_block 🦋 on b33fc556fa304945d7ec0c7ea2594acc32e3ff6e
Thanks Andrew!

**Review by andyg0808 — APPROVED** [review]

**Comment by andyg0808** [comment]
CR :dog: through b33fc55

**Comment by jordycosta** [comment]
Getting a serverless function 500 error when viewing this preview

https://react-commerce-prod-git-tools-hide-facets-on-tool-cc49fd-ifixit.vercel.app/Tools

**Comment by rickisXP** [comment]
It looks like the preview is throwing 500s on `/Parts` and `/Tools`

**Comment by deltuh-vee** [comment]
dev_block ✊

**Comment by sumukhipandey** [comment]
un_dev_block 🦋 Merged main
[Preview ](https://react-commerce-git-tools-hide-facets-on-tools-children-ifixit.vercel.app/Parts)should now be fine

**Comment by sumukhipandey** [comment]
dev_block 🦋

**Comment by sumukhipandey** [comment]
un_dev_block 🦋

**Review by Michelle5102 — APPROVED** [review]
CR :octocat:

**Comment by rickisXP** [comment]
QA :penguin: but deploy_block :question: on a concern
- All Tools page only has `Tool Category` and `Price Range` filters :heavy_check_mark: 
- Facets appear as expected :heavy_check_mark: 
- On a child tools page, the `Worksin` filter is available to use. Is this filter supposed to be shown? :warning: 

image

**Comment by sumukhipandey** [comment]
> QA 🐧 but deploy_block ❓ on a concern
> 
> * All Tools page only has `Tool Category` and `Price Range` filters ✔️
> * Facets appear as expected ✔️
> * On a child tools page, the `Worksin` filter is available to use. Is this filter supposed to be shown? ⚠️
> 
> image

un_deploy_block 🦋 That has been addressed [here](https://github.com/iFixit/react-commerce/pull/515)

**Pull request merged**